### PR TITLE
RADIUS container fixes

### PR DESCRIPTION
**Pull request by yaleman**
Fixes the ongoing issues with container builds.

- Moves the RADIUS container to `freeradius/freeradius` which is based on Ubuntu and is regularly updated.
- Makes an error message in the RADIUS module nicer
- Allows scripted password auth via a secret env var because I'm a bad person and wanted to script server setup.
- Fixes some CLI i/o because I'm a worse person.

- [x] cargo fmt has been run
- [x] cargo clippy has been run
- [x] cargo test has been run and passes

### CODE REVIEW

File: rlm_python/Dockerfile [hunk 1]
```diff
@@ -1,33 +1,21 @@
...
-    zypper mr -k repo-oss && \
-    zypper mr -k repo-update
-
-# ======================
-FROM repos
-
+FROM freeradius/freeradius-server:latest
 EXPOSE 1812 1813
+ARG RADIUS_USER=freerad
+ENV TZ=Etc/UTC
```
Comment (Firstyear) [inline rlm_python/Dockerfile:4]: I think we shouldn't embed the TZ at build here, but recommend people bind mount it in from the container host `"/etc/localtime:/etc/localtime:ro"`

File: tools/cli/src/opt/kanidm.rs [hunk 2]
```diff
@@ -500,7 +503,11 @@ pub struct LoginOpt {
     #[clap(flatten)]
     copt: CommonOpt,
     #[clap(short, long)]
+    /// Use a passkey for authentication
```
Comment (Firstyear) [inline tools/cli/src/opt/kanidm.rs:506]: The webauthn field is never used so I think we can remove it actually ....

File: tools/cli/src/cli/session.rs [hunk 3]
```diff
@@ -228,9 +238,13 @@ impl LoginOpt {
 
     pub async fn exec(&self) {
         let mut client = self.copt.to_unauth_client();
-
-        // TODO: remove this anon, nobody should do default anonymous
-        let username = self.copt.username.as_deref().unwrap_or("anonymous");
+        let username = match self.copt.username.as_deref() {
+            Some(val) => val,
+            None => {
+                error!("Please specify a username to login with!");
```
Comment (Firstyear) [inline tools/cli/src/cli/session.rs:244]: Should say "specify a username with `-D` to login"

**Review by Firstyear — CHANGES_REQUESTED** [review]

File: rlm_python/Dockerfile (hunk 1 above)
Comment (yaleman) [reply rlm_python/Dockerfile:4]: It's an unavoidable part of the build process - if you install tzdata, you have to select a timezone and there's no other way of fixing it 😡

Comment (Firstyear) [reply rlm_python/Dockerfile:4]: :(

File: tools/cli/src/cli/session.rs (hunk 3 above)
Comment (Firstyear) [reply tools/cli/src/cli/session.rs:244]: Did you push the update with this in it?

Comment (yaleman) [reply tools/cli/src/cli/session.rs]: I have now!

**Review by Firstyear — APPROVED** [review]

**Pull request merged**